src/utils/open-browser.js

```diff
@@ -33,4 +33,4 @@ const openBrowser = async function ({ silentBrowserNoneError, url }) {
   }
 }
 
-module.exports = openBrowser
+module.exports = { openBrowser }
```

Comment:
The following consumers should be updated with the new signature:

https://github.com/netlify/cli/blob/9165fe3e69b69dab015fdd930feed8ebec89181a/src/commands/addons/auth.js#L4

https://github.com/netlify/cli/blob/9165fe3e69b69dab015fdd930feed8ebec89181a/src/commands/deploy.js#L24

https://github.com/netlify/cli/blob/9165fe3e69b69dab015fdd930feed8ebec89181a/src/commands/dev/index.js#L20

https://github.com/netlify/cli/blob/9165fe3e69b69dab015fdd930feed8ebec89181a/src/commands/open/admin.js#L3

https://github.com/netlify/cli/blob/9165fe3e69b69dab015fdd930feed8ebec89181a/src/commands/open/site.js#L3

https://github.com/netlify/cli/blob/9165fe3e69b69dab015fdd930feed8ebec89181a/src/utils/command.js#L14

https://github.com/netlify/cli/pull/3592/files#diff-9f1c90587f2f1996609d5d02f01ef29eba18d163b06171d7b11a651376e40461R7

Reply:
Thx have overseen that 👍